source/cloud-native-platform/troubleshooting/index.html.md.erb

```diff
@@ -505,3 +505,39 @@ Check the pipeline passed after merging. If the pipeline failed, it's possible t
 Check the IP your rule is forwarding to. It should be the private IP of the frontend Application Gateway.
 
 You can find this [here](https://github.com/hmcts/azure-platform-terraform/blob/6f0b867e75b7e9cee9e7adc87084f6911eb5373d/environments/sbox/sbox.tfvars#L20).
+
+### Java (Spring Boot) Golden Path Errors
+
+#### - Suppress false positives in the OWASP Dependency Checker
+
+The [OWASP Dependency Checker](https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/index.html) provides monitoring of the project's dependencies and creates a report of known
+vulnerable components that are included in the build. The checker plugin is declared in the Golden path for Java apps. Because of the way the dependency checker works, false positives and false negatives may exist. We can surepress these false positives by providing
+the dependency checker with the path to a suppression file in the `build.gradle` file.
+
+```
+dependencyCheck {
+  suppressionFile = 'path/to/supression.xml'
+}
+```
+#### - "NoSuchMethodError" when running the OWASP Dependency Checker
+
+With the Dependency-Check v9.0.0 users may encounter issues with  `NoSuchMethodError` exceptions due to dependency resolution.
+
+#### Solution
+
+You will need to pin some of the transitive dependencies to the versions that are compatible with the Dependency-Check. e.g.
+
+``` groovy
+dependencies {
+    constraints {
+        // org.owasp.dependencycheck needs at least this version of jackson. Other plugins pull in older versions..
+        add("implementation", "com.fasterxml.jackson:jackson-bom:2.16.1")
+
+        // org.owasp.dependencycheck needs these versions. Other plugins pull in older versions..
+        add("implementation", "org.apache.commons:commons-lang3:3.14.0")
+        add("implementation", "org.apache.commons:commons-text:1.11.0")
+    }
+}
+```
+
```

Comment:
Looks good, can you maybe add something similar to [this](https://github.com/hmcts/hmcts.github.io/blob/2d8e6b22384916b8199941b77a24ab9f805ddb29/source/cloud-native-platform/troubleshooting/index.html.md.erb#L400-L402) that links to mvnrepository so people know where they can find updated version numbers?

And maybe something about `./gradlew dependencies` similar to [this](https://github.com/hmcts/hmcts.github.io/blob/2d8e6b22384916b8199941b77a24ab9f805ddb29/source/cloud-native-platform/troubleshooting/index.html.md.erb#L394).